tox.ini

```diff
@@ -1,31 +1,20 @@
 [tox]
-#   PyPy3 support pending a release of a fix for
-#   https://bitbucket.org/pypy/pypy/issue/1946)
-#envlist = py27,py33,py34,pypy,pypy3
-envlist = py27,py33,py34,py35,pypy
+envlist = py27,py34,py35,py36,py37,pypy,pypy3,coverage
 
 [testenv]
 commands =
-    python setup.py -q test -q
-# without explicit deps, setup.py test will download a bunch of eggs into $PWD
+    zope-testrunner --test-path=src []
 deps =
-    zc.customdoctests
-    zope.testing
-    zope.testrunner
-    manuel
-    mock
+    .[test]
 
 [testenv:coverage]
 usedevelop = true
 basepython =
-    python2.7
-setenv =
-    COVERAGE_HOME={toxinidir}
-    COVERAGE_PROCESS_START={toxinidir}/.coveragerc
+    python3.6
 commands =
-    coverage run --source=zdaemon setup.py -q test
+    coverage run -m zope.testrunner --test-path=src []
     coverage combine
-    coverage report
+    coverage report --fail-under=65
```

Comment:
I have doubts about this.

COVERAGE_PROCESS_START and the `parallel=true` bit in `.coveragerc` seem to indicate this was geared up to track multi-process coverage (zdrun.py has a couple of `os.fork()`s).

If I run `tox -e coverage` today, I see 78% coverage.  It seems that your changes break the above setup and therefore coverage drops.

Reply:
I did not fully understand the interaction of `COVERAGE_PROCESS_START` and coverage on sub-processes. Thanks for clarifying that.